core/domain/exp_jobs_one_off.py

```diff
@@ -470,3 +473,61 @@ def map(item):
     @staticmethod
     def reduce(key, values):
         yield (key, values)
+
+
+class ExplorationContentValidationJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that checks the html content of an exploration and validates it.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+
+        exploration = exp_services.get_exploration_from_model(item)
+        err_dict = {}
+
+        with open('./core/domain/htmlValidation.yaml', 'r') as fp:
+            htmlValidation = yaml.load(fp)
+            allowed_parent_list = htmlValidation['allowed_parent_list']
+            allowed_tag_list = htmlValidation['allowed_tag_list']
+
+        for state in exploration.states.itervalues():
+            result = exp_services.find_all_values_for_key(
```

Comment:
Does this process also account for nested rte content? Such as content inside tabs.

Reply:
Yes.